recipes/systemc-cci/1.0.0/patches/rapidjson.patch

```diff
@@ -0,0 +1,58 @@
+diff --git a/packages/rapidjson/include/rapidjson/document.h b/packages/rapidjson/include/rapidjson/document.h
```

Comment:
Any chances to use rapidjson from CCI?

Reply:
Dont think so unless the complete SystemC CCI library is reworked.
I assume they had a good reason to include the source.

Reply:
In most cases, the reason is ease of use for non-package manager users.
Which usually are Windows users.